pkg/controller/daemon/daemon_controller.go

```diff
@@ -850,7 +857,7 @@ func (dsc *DaemonSetsController) podsShouldBeOnNode(
 		// If daemon pod is supposed to be running on node, but more than 1 daemon pod is running, delete the excess daemon pods.
 		// Sort the daemon pods by creation time, so the oldest is preserved.
```

Comment:
nit: scheduled pod is preserved first; if more than one pod can be preserved, the oldest pod is preserved.